### PR TITLE
Add status metrics, grafana panels about S3 GC

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflash/issues/6827, close https://github.com/pingcap/tiflash/issues/7258

Problem Summary:

### What is changed and how it works?

* change default value of `storage.cache.delta_rate` from 0.3 to 0.1 
* Refine `AsynchronousMetrics::getPageStorageFileUsage` under different non-disagg/disagg mode
* Show the number of keys/number of bytes of different kind of keys when uploading checkpoint to remote store
* Try set the lifecycle rule and set a metrics to Grafana (seems it is still failed, will investigate why in later PRs)
* Add metrics about which node is the GCOwner, whether S3GCManager is running

image
image
image

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/S3/S3Common.cpp [hunk 1]
```diff
@@ -712,17 +718,9 @@ void ensureLifecycleRuleExist(const TiFlashS3Client & client, Int32 expire_days)
...
-    else
-    {
-        UNUSED(expire_days);
-        LOG_WARNING(client.log, "The lifecycle rule with filter \"{}\" has not been set, please check the bucket lifecycle configuration", TaggingObjectIsDeleted);
-        return;
+        return true;
     }
 
-#if 0
-    // Adding rule by AWS SDK is failed, don't know why
```
Comment (JaySon-Huang) [inline dbms/src/Storages/S3/S3Common.cpp:725]: This PR each time will try and log a warnings each time S3GC is run. If the check fail, just log a warning and continue the S3 GC process.
Adding rule is still rejected by AWS, will find out why in later PRs.

**Review by lidezhu — APPROVED** [review]
LGTM

**Review by CalvinNeo — APPROVED** [review]
lgtm

**Review by flowbehappy — APPROVED** [review]
LGTM

**Pull request merged**